rest_framework/compat.py

```diff
@@ -122,6 +122,20 @@ def __init__(self, *args, **kwargs):
             self.message = kwargs.pop('message', self.message)
             super(MaxLengthValidator, self).__init__(*args, **kwargs)
 
+# ForeignKey's to_fields parameter was changed in Django 1.6 to an array named to_fields
+# to_fields is an array but django lets you only set one to_field for ForeignKeys
+# https://docs.djangoproject.com/en/1.7/ref/models/fields/#django.db.models.ForeignKey.to_field
+# see also ForeignKey and ForeignObject in
+# https://docs.djangoproject.com/en/1.7/_modules/django/db/models/fields/related/
+
+if django.VERSION >= (1, 6):
```

Comment:
This is the minimum supported version in master, so oughta be able to drop the compat bit.